protocols/braintrust/overview.md

```diff
@@ -0,0 +1,62 @@
+# Braintrust
+
+## Overview
+Braintrust is the first decentralized talent network that connects skilled, vetted knowledge workers with the world’s most reputable brands. The community that relies on Braintrust to find work are the same people who own and build it, ensuring the network always serves the needs of its users, instead of a centrally-controlled corporation. Unlike other marketplaces that take 20% to 40% of talent earnings, Braintrust allows talent to keep 100% of earnings and enables clients to make their budgets go further by cutting out the unnecessary middle men.
+The community expands beyond 50,000 members across 100+ countries. Braintrust partners with global enterprises including Nestle, Porsche, Atlassian, Goldman Sachs, and Nike. Braintrust is backed by Tiger Global, Coinbase Ventures, Pantera, and Multicoin Capital along with a contingent of individual investors. 
+
+
+## Governance
+
+### The Token-Enabled BTRST Governance Model 
```

Comment:
👀

Reply:
Demo of markdown formatting https://hackmd.io/JSKzs_V0R--TP7bmHstn0w